fusor-ember-cli/app/controllers/assign-nodes.js

```diff
@@ -26,11 +26,13 @@ export default Ember.Controller.extend({
   }.property('model.images.[]'),
 
   unassignedRoles: function() {
-    var unassignedRoles = [];
+    var unassignedRoles = Ember.A();
     var params = this.get('model.plan.parameters');
     var self = this;
+    var value = null;
     this.get('model.plan.roles').forEach(function(role) {
-      if ( self.getParamValue(role.get('flavorParameterName'), params) == null ) {
+      value = self.getParamValue(role.get('flavorParameterName'), params);
+      if (value === 'baremetal' || Ember.isNone(value)) {
```

Comment:
@sdherr, the response in the **unassigned** json shows that the value is `baremetal` rather than `null`. Is this a backend change? If correct, then this fix shows the unassigned roles.